cookbook/deployment/deploying_workflows.py

```diff
@@ -0,0 +1,210 @@
+"""
+Deploying Workflows - Registration
```

Comment:
Hey @cosmicBboy! This page isn't needed anymore.

Reply:
nice catch!